Modules/Filtering/DistanceMap/include/itkDirectedHausdorffDistanceImageFilter.hxx

```diff
@@ -35,6 +35,8 @@ DirectedHausdorffDistanceImageFilter<TInputImage1, TInputImage2>::DirectedHausdo
   this->SetNumberOfRequiredInputs(2);
 
   m_DistanceMap = nullptr;
+  m_MaxDistance = NumericTraits<RealType>::ZeroValue();
+  m_PixelCount = 0;
```

Comment:
Please consider C++11 in-class member initialization (in its header file), instead. An empty pair of curly braces `{}` should be sufficient:
  
    RealType       m_MaxDistance{};
    IdentifierType m_PixelCount{};